test/data/eval/success/examples/example20.kriti

```diff
@@ -0,0 +1,7 @@
+[
+  {{ $[0]?.notIndex }},
+  {{ $[0]?.['notIndex'] }},
+  {{ $?.[9999] }},
+  {{ $?.[9999].index }},
+  {{ $[0]?.notIndex.baz }}
+]
```

Comment:
Highlighting this test as examples of the syntax.

Reply:
Looks very straightforward.